app/application/views/onboarding_3_enter_details_form.php

```diff
@@ -44,6 +44,9 @@
                         <div id="volunteering">
                             <h1>Enter Your Details</h1>
 
+                            <h4><br /></h4>
```

Comment:
There's no need to wrap `<br>` in `<h4>`.

Also, there's no need to end a `br` with a `/`. It's an optional component of the spec.